newswires/client/src/App.tsx

```diff
@@ -81,6 +105,36 @@ export function App() {
 					max-height: 100vh;
 				`}
 			>
+				{displayDisclaimer && (
+					<EuiModal
+						aria-labelledby={'Use with caution'}
```

Comment:
If we're setting the aria label inline then the property should be `aria-label` instead. I'd be inclined to keep it as `aria-labelledby` though, and generate an id for the EuiModalHeaderTitle that can be passed as the value, so that the aria label will automatically match the value of the header title